ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles.json

```diff
@@ -4,21 +4,28 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "RoleName": "hmcts-admin",
     "ReadOnly": "Y",
-    "AccessProfiles": "GS_profile"
+    "AccessProfiles": "GS_profile,caselinks_admin_profile"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "RoleName": "hmcts-judiciary",
     "ReadOnly": "Y",
-    "AccessProfiles": "GS_profile"
+    "AccessProfiles": "GS_profile,caselinks_profile"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "RoleName": "hmcts-staff",
```

Comment:
Looks like it talks about this role in the guide - is it really needed though? think it might need set up in AM if so?